apps/web/screens/components/ConnectionCard.tsx

```diff
@@ -114,12 +110,12 @@ export function ConnectionCard({connection: conn}: ConnectionCardProps) {
                       console.error('syncConnection error', err)
                     })
                 }>
-                <Play />
                 Full sync
               </DropdownMenuItem>
 
+              {/* TODO: Add a confirmation dialog */}
```

Comment:
Note: we need a confirmation dialog for deleting a connection

Reply:
Yes, and probably language to help you understand what actually happens when you do.

Reply:
Do we have a task for it?

Reply:
Yes, I'll take this one